app/src/organisms/ProtocolUpload/hooks/useMostRecentRunId.ts

```diff
@@ -1,8 +1,9 @@
 import { useAllRunsQuery } from '@opentrons/react-api-client'
+import { last } from 'lodash'
```

Comment:
Basically we import a specific one instead of importing the entire `lodash`

```suggestion
import last from 'lodash/last'
```